dataactvalidator/filestreaming/loadFile.py

```diff
@@ -28,20 +28,25 @@ def loadDomainValues(basePath, localSFPath = None, localProgramActivity = None):
     print("Loading object class")
     loadObjectClass(os.path.join(basePath,"object_class.csv"))
 
-    # SF 133 is kept on S3, so need to download that
-    reader = CsvS3Reader()
+    # Since we're temporarily bulk loading the historical SF-133 data,
+    # we'll also need to temporarily bypass the current SF-133 loader,
+    # which will delete everything in that table whenever we deploy.
+
+    # # SF 133 is kept on S3, so need to download that
+    # reader = CsvS3Reader()
```

Comment:
I know this is commented and not used right now, but the reader should be initialized in the `else` because at that point it is confirmed that it's not coming from a local path but instead from S3